ops-bedrock/Dockerfile.stateviz

```diff
@@ -2,18 +2,19 @@ FROM golang:1.20.4-alpine3.16 as builder
 
 RUN apk add --no-cache make gcc musl-dev linux-headers git jq bash
 
+COPY ./vendor /app/vendor
 COPY ./go.mod /app/go.mod
 COPY ./go.sum /app/go.sum
 
 COPY ./op-bindings /app/op-bindings
 COPY ./op-service /app/op-service
 
 WORKDIR /app/op-node
-RUN go mod download -x
+# RUN go mod download -x
 
 COPY ./op-node /app/op-node
 
-RUN go build -o ./bin/stateviz ./cmd/stateviz
+RUN --mount=type=cache,target="/root/.cache/go-build" go build -o ./bin/stateviz ./cmd/stateviz
```

Comment:
I am curious what's the purpose of those changes?

Reply:
streamlines the build, I removed it for now